plugins/analytics/public/dockerhub.py

```diff
@@ -0,0 +1,188 @@
+import json
+import logging
+import re
+
+import requests
+from core import taskmanager
+from core.config.config import yeti_config
+from core.schemas import observable, task
+from core.schemas.observable import Observable, ObservableType
+
+
+class DockerHubAPI(object):
```

Comment:
`Class DockerHubAPI:` is better

Reply:
Fixed in 65c30d39a9d18f5b8fa26d20d89e66a6a0c90091